server/data/content_types.json

```diff
@@ -204,5 +204,171 @@
             "body_html": {"formatOptions": ["h1", "h2", "bold", "italic", "underline", "quote", "anchor"]},
             "abstract": {"formatOptions": ["h1", "h2", "bold", "italic", "underline", "quote", "anchor"]}
         }
+    },
+    {
+        "_id": "news",
+        "label": "News",
+        "priority": 30,
+        "enabled": true,
+        "schema": {
+            "anpa_category": {},
+            "urgency": {"nullable": true},
+            "subject": {},
+            "slugline": {"required": true},
+            "ednote": {},
+
+            "headline": {"required": true},
+            "body_html": {"required": true},
+            "abstract": {"required": true},
+            "byline": {},
+            "feature_image": {"type": "picture", "required": true},
+
+            "topic_news": {"required": true},
+            "age_range": {},
+            "territory": {"required": true},
+            "reader_type": {},
+            "assets": {},
+            "relatedItems": {}
```

Comment:
Some of these items in the schema need to have translations on the client ([see here](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/superdesk-workspace/content/content.js#L533-L555)). For example, `relatedItems` and `assets` isn't there at the moment. 

Unfortunately, this is not a very maintainable solution and a better one would be required. I would suggest adding another field in the backend, named `label`, which would describe what the editor should show. Any other suggestions or solutions are welcome.